src/vivarium_public_health/disease/model.py

```diff
@@ -109,7 +112,21 @@ def load_population_columns(self, pop_data):
 
         assert self.initial_state in {s.state_id for s in self.states}
 
-        state_names, weights_bins = self.get_state_weights(pop_data.index)
+        # FIXME: this is a hack to figure out whether or not we're at the simulation start based on the fact that the
```

Comment:
Make a ticket for this fixme.  I think what we should do is always provide the user_data with a key indicating something about the state of the system when people are being created.  It should be a relatively small fix (one change in the engine, and the rest in the fertility component).

Reply:
ticket created